src/public/app/widgets/toc.js

```diff
@@ -227,15 +232,15 @@ export default class TocWidget extends RightPanelWidget {
                 // Scroll to the end of the note to guarantee the
                 // next scroll is a backwards scroll that places the
                 // caret at the top of the screen
-                model.change(writer => {
-                    writer.setSelection(root.getChild(root.childCount - 1), 0);
-                });
-                textEditor.editing.view.scrollToTheSelection();
-                // Backwards scroll to the heading
-                model.change(writer => {
-                    writer.setSelection(headingNode, 0);
-                });
-                textEditor.editing.view.scrollToTheSelection();
+                // model.change(writer => {
+                //     writer.setSelection(root.getChild(root.childCount - 1), 0);
+                // });
+                // textEditor.editing.view.scrollToTheSelection();
+                // // Backwards scroll to the heading
+                // model.change(writer => {
+                //     writer.setSelection(headingNode, 0);
+                // });
+                // textEditor.editing.view.scrollToTheSelection();
```

Comment:
If this is indeed not needed then better to simply delete the code.

Reply:
I found that adding css: 
div.component.scrolling-container{
    scroll-behavior: smooth;
}
can also achieve smooth scrolling, and at the same time make the search function scroll smoothly. I withdraw this pr and modify the css.